perf/CMakeLists.txt

```diff
@@ -98,3 +123,15 @@ add_custom_target(test-perf
                   DEPENDS test-c-perf test-lua-perf
                   COMMENT "Running performance tests"
 )
+
+
+set(PERF_SUMMARY ${PERF_OUTPUT_DIR}/summary.txt)
+add_custom_target(test-perf-aggregate
+                  DEPENDS ${BENCH_RESULTS}
+                  BYPRODUCTS ${PERF_SUMMARY}
+                  COMMENT "Aggregate performance test results"
```

Comment:
I propose to print the path to a final report.

Reply:
Good idea!
Added to the comment as you suggested.